api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3127,20 +3127,25 @@ public void savePatientIdentifierType_shouldThrowErrorWhenTryingToSaveAPatientId
 		ps.savePatientIdentifierType(pit);
 	}
 	
-	@Test(expected = PatientIdentifierTypeLockedException.class)
```

Comment:
Why  modify this ? Is it any better?

Reply:
I removed the expected attribute in this case because I wanted some uniformity in the tests i was writing( I removed the expected attribute from the next test i wrote). In this particular test, it does not really make much difference but in the next test, I replaced the expected attribute with `expectedException.expectMessage("A reason is required when retiring an identifier type");
` so that we are sure of the exact error message that is thrown and thus we are sure that it is the exact error that has occurred. Its kind of like an assert statement. Using the expected attribute just states the type of error, but I was looking for specifics